src/models/d2Model.ts

```diff
@@ -173,6 +173,7 @@ export class DataElementModel extends D2Model {
         "dataSets",
         "legendSets",
         "optionSets",
+        "optionSets.options",
```

Comment:
Maybe this?

Reply:
Already in devel -> https://github.com/EyeSeeTea/metadata-synchronization/blob/development/src/models/dhis/metadata.ts#L81